src/js/components/tasks/detail.tsx

```diff
@@ -201,8 +202,22 @@ const TaskDetail = (props: RouteComponentProps) => {
     const captureButtonAction = () => {
       /* istanbul ignore else */
       if (devOrg) {
-        setFetchingChanges(true);
-        doRefetchOrg(devOrg);
+        let shouldCheck = true;
+        const checkAfterMinutes = window.GLOBALS.ORG_RECHECK_MINUTES;
+        if (
+          devOrg.last_checked_unsaved_changes_at !== null &&
+          typeof checkAfterMinutes === 'number'
+        ) {
+          const lastChecked = parseISO(devOrg.last_checked_unsaved_changes_at);
+          const shouldCheckAfter = addMinutes(lastChecked, checkAfterMinutes);
+          shouldCheck = isPast(shouldCheckAfter);
+        }
+        if (devOrg.has_unsaved_changes && !shouldCheck) {
+          setCaptureModalOpen(true);
+        } else {
+          setFetchingChanges(true);
+          doRefetchOrg(devOrg);
+        }
```

Comment:
@wlonk It was pretty straightforward to implement this check on the front end, so I did that here. If we want to avoid this duplication though, we could do that by having the back end view return a different response to indicate to the front end whether or not changes are actually being fetched. I don't feel strongly either way -- happy for your suggestion.

Reply:
I'm OK with this, it's just fine.